project/eScavenge/views.py

```diff
@@ -70,3 +71,64 @@ def teamPage(request):
     teamhistory = userpage.history.all()
     context = {'team': userpage, 'teamlist': teamlist, 'teamhistory': teamhistory, 'command': command}
     return render(request, 'teamPage.html', context)
+
+def gamemakerPage(request):
+    username = request.session.get('username')
+    message = ''
+    cli = CLI(COMMANDS)
+    if username != GM.username:
+        return HttpResponseForbidden()
+    if (request.method == 'POST'):
+        if (request.POST.get("selectGame")):
+            request.session['curgame'] = request.POST["gameObject"]
+    if('curgame' in request.session.keys()):
+        curgame = request.session.get('curgame')
+        try:
+            gameteam = Game.objects.get(name = curgame).teams.all()
+        except:
```

Comment:
You should restrict your `except` line to the specific exception you're expecting. In this case I would guess you're expecting Game.DoesNotExist so it would be `except Game.DoesNotExist:`